### ISSUE TITLE
Mumble Server Not Installing

### ISSUE THREAD

**Issue by NetlyLLC** [pterodactyl/panel#4006]
### Current Behavior

When I go to the admin dashboard, and create a new server using the Mumble egg, it does not install. When I start the server, it says a file is missing
![image](https://user-images.githubusercontent.com/46270991/158701806-5b980f4a-b7ab-4863-8a9b-ef3a6314b8f7.png)

### Expected Behavior

When I create the server, it creates the files it needs, and starts normally.

### Steps to Reproduce

Go to the admin dashboard, click servers, click create new, choose the details you want, make sure that it is on Voice servers, mumble, and  that Skip Egg Install script is disabled, so that it creates the server. Wait for it to "install", then try starting the server.

### Panel Version

1.7.0

### Wings Version

1.6.1

### Games and/or Eggs Affected

Mumble Server

### Docker Image

ghcr.io/pterodactyl/yolks:alpine

### Error Logs

```bash
Wings: https://ptero.co/ymujygawuz
panel: https://bin.ptdl.co/oky6q/
```

### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.
- [X] I have provided all relevant details, including the specific game and Docker images I am using if this issue is related to running a server.
- [X] I have checked in the Discord server and believe this is a bug with the software, and not a configuration issue with my specific system.

**Comment by parkervcp** [comment]
This issue was called out here - https://github.com/parkervcp/eggs/issues/1535

As the mumble guys changed their packaging it broke the install script.

**Comment by NetlyLLC** [comment]
Will it ever be fixed?

**Comment by parkervcp** [comment]
It will be. We've been busy.

**Comment by koutsie** [comment]
ouch.

**Comment by CalMatar** [comment]
Is there an ETA on this fix as this issue is now eight months old, or a workaround to make it work fairly easily?

**Comment by fucksophie** [comment]
Hello! Do you guys plan on doing anything with this? I bet you could just actually use the debian yolk + `sudo apt-get install mumble-server`? Or just build mumble-server? I'm very inexperienced with eggs and I couldn't figure out how to do that. Thank you!

**Comment by ghost** [comment]
I am also having issues with this do you guys ever plan to fix it?

**Comment by parkervcp** [comment]
If you check PR's #4437 is done and just needs to be merged

**Comment by CalMatar** [comment]
So once the testing on the databases and the UI is done this should be merged and a pterodactyl update should be forthcoming? Is that what I am understanding from that?

**Comment by trenutoo** [comment]
Fixed in #4437 v1.11.0
